### PR TITLE
[NIFI-2038] [NIFI-2144] [NIFI-2031] [NIFI-2036] [NIFI-2037]

### PR DESCRIPTION
**Pull request by scottyaslan**

### CODE REVIEW

**Comment by moranr** [comment]
For **NIFI-2038**
* For #header .component-button, height and width properties are declared twice at 40px and 56px. Should only be 56px
* Icons (#header .icon and #header .fa) should be 32px (​​NIFI-2037)
* Looks like #header .icon ​will need padding-left:6px added to center them within the button​s (once size is updated to 32px)​
* The version of each icon with a plus should display on hover. Currently it only shows once you start dragging
* When button is click and while dragging, set cursor to cursor:grabbing

See mockup as example (nifi-drag-drop-component.png)
nifi-drag-drop-component

**Comment by moranr** [comment]
**NIFI-2144** looks good – seeing the correct icons in place.

**Comment by moranr** [comment]
​**NIFI-2031** looks good

For **NIFI-2036,** NiFi logo seems to sit a little low after the other toolbar changes. Please *change* `margin:0 20px 0 15px;` *to* `margin:0 20px 3px 15px;`

​For **NIFI-2037** `#header .icon` and `#header .fa` should be `font-size:32px`

**Comment by scottyaslan** [comment]
@moranr I have addressed all comments and updated the PR.

@mcgilman ready for review please sir

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/header.css [hunk 1]
```diff
@@ -34,12 +34,13 @@ md-toolbar.md-small .md-toolbar-tools {
 }
 
 #header .icon {
-    font-size:30px;
+    font-size:32px;
     color: #004849;
+    padding-left:6px
```
Comment (moranr) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/header.css]: @scottyaslan looks like the 6px padding-left was not enough to center the icons within each button. I must have misjudged it originally. Changing it to 12px works for me now.

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/controllers/nf-ng-canvas-toolbox-controller.js [hunk 2]
```diff
@@ -94,12 +94,21 @@ nf.ng.Canvas.ToolboxCtrl = function (processorComponent,
...
+                component.getElement().removeClass(component.hoverIcon).addClass(component.icon);
+            })
+
             return {
                 zIndex: 1011,
                 revert: true,
                 revertDuration: 0,
                 cancel: false,
                 containment: 'body',
+                cursor: 'grabbing',
```
Comment (moranr) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/controllers/nf-ng-canvas-toolbox-controller.js]: @scottyaslan I think we can add `cursor: '-moz-grabbing'` and `cursor: '-webkit-grabbing'` vendor prefixes here to get the cursor to show on non-supported browsers.

Comment (scottyaslan) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/controllers/nf-ng-canvas-toolbox-controller.js]: I can try this...but I don't think jQuery UI will like that....it is not like CSS in this case

**Comment by moranr** [comment]
Everything is looking good!

**Comment by mcgilman** [comment]
Looks good @scottyaslan! This has been merged to master. Thanks!

**Pull request closed without merging**